src/main/java/mate/academy/dao/impl/OrderDaoImpl.java

```diff
@@ -39,7 +39,7 @@ public Order add(Order order) {
     @Override
     public List<Order> getByUser(User user) {
         try (Session session = HibernateUtil.getSessionFactory().openSession()) {
-            Query<Order> query = session.createQuery("from Order o "
+            Query<Order> query = session.createQuery("select distinct from Order o "
```

Comment:
please provide the screenshot that it works properly

Reply:
https://prnt.sc/25z2dsc